alacritty/src/event.rs

```diff
@@ -393,9 +404,6 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
         self.search_state.origin = if self.terminal.mode().contains(TermMode::VI) {
             self.terminal.vi_mode_cursor.point
         } else {
-            // Clear search, since it is used as the active match.
-            self.terminal.selection = None;
-
```

Comment:
I'm not sure if we should keep it tbqh, I like always clearing a selection, since selection could interfere with focused match and you won't see a focused match when you start searching due to selection.